cypress/features/regression/designSystem/menuDefaultDivider.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Design Systems Menu component - divider story
+  I want to check Design Systems Menu component default divider story properties
+
+  Background: Open Design Systems Menu component default divider page
+    Given I open design systems default_divider "Menu" component in no iframe
+
+  @positive
+  Scenario: Check the persistence of Menu component
+    # When I open design systems default "Menu" component in no iframe
```

Comment:
please add one comment before this one, why it's commented

Reply:
updated